test/plugins/trigger.test.js

```diff
@@ -2034,6 +2036,88 @@ describe('trigger tests', () => {
         assert.equal(upstreamPipeline.getBuildsOf('target').length, 2);
     });
 
+    it('[ ~sd@2:a ] and [ ~sd@3:a ], [ ~sd@2:a ] is triggered and [ ~sd@3:a ] is not triggerd when sd@3 pipeline is deleted from DB', async () => {
```

Comment:
I think these tests should be contained in `builds.test.js` since it seems better to throw exception explicitly.

Ex.
https://github.com/screwdriver-cd/screwdriver/blob/ec99cc8b486bf71a85676322715c8559df52eb9a/test/plugins/builds.test.js#L5320

I do not think it is good to put any tests in `trigger.test.js`.

Reply:
I fixed it!